src/components/codeSnippet/codeSnippet.tsx

```diff
@@ -0,0 +1,58 @@
+import './codeSnippet.css'
+import React, { useEffect, useState } from 'react'
+import { EditorView, minimalSetup } from 'codemirror'
+import { EditorState } from '@codemirror/state'
+import { Box } from '@mui/system'
+import { Typography } from '@mui/material'
+import { v4 as getUUID } from 'uuid'
+import { languages } from '@codemirror/language-data'
+
+export interface CodeSnippetProps {
+  /** Code that will be displayed. */
+  code: string
+  /** Language type needs to be provided so that the right language extension can be used to format code. Language is case-sensitive. Please refer to the [supported languages list](https://codemirror.net/5/mode/). */
+  language: string
```

Comment:
```suggestion
  /** Language type needs to be provided so that the right language extension can be used to format code. Please refer to the [supported languages list](https://codemirror.net/5/mode/). */
  language: string
```